src/popup.js

```diff
@@ -16,58 +16,16 @@
  * You should have received a copy of the GNU General Public License
```

Comment:
it'd be cool if all of this code could be shared between options and popup
refactor 2.0